definitions/npm/redux-reactors_v1.x.x/test_redux-reactors_v1.x.x.js

```diff
@@ -0,0 +1,117 @@
+// @flow
+
+import { describe, it } from 'flow-typed-test';
+import { reactorEnhancer, createReactor } from 'redux-reactors';
+import type { ReactorAction } from 'redux-reactors';
+
+/* Helper types from 'redux'
```

Comment:
Because they're copied in the implementation, you don't need to copy them again here.

Reply:
Good call!  Updated.